src/components/AppContent/AppContent.vue

```diff
@@ -24,14 +24,37 @@
 
 <template>
 	<main id="app-content-vue" class="app-content no-snapper">
-		<!-- @slot Provide content to the app content -->
-		<slot />
+		<div id="app-content-wrapper" :style="`height: ${contentHeight}px;`">
+			<!-- @slot Provide content to the app content -->
+			<slot />
+			<Splitpanes
+				class="default-theme"
+				@resized="handlePaneResize">
+				<Pane class="sidebar"
+					:size="paneSizes.sidebar || paneDefaults.sidebar.size"
+					:min-size="paneDefaults.sidebar.min"
+					:max-size="paneDefaults.sidebar.max">
+					<slot name="sidebar" />
+				</Pane>
```

Comment:
Sidebar is what we have on the far right of the apps, please use something like `list` or something?

Also, what happens if you don't  have a list? Do we really need the Pane component to be used in that case?
Is it ignore? Maybe it would be better to have a v-if and v-else if there are no list slot to just output the main content?